tests/loader_get_proc_addr_tests.cpp

```diff
@@ -60,10 +60,17 @@ TEST(GetProcAddr, GlobalFunctions) {
         auto CreateInstance = reinterpret_cast<PFN_vkCreateInstance>(gipa(NULL, "vkCreateInstance"));
         handle_assert_has_value(CreateInstance);
     }
+#if !defined(VULKANSC)
     // Now create an instance and query the functions again - should work because the instance version is less than 1.2
     for (int i = 0; i <= 2; i++) {
         InstWrapper inst{env.vulkan_functions};
         inst.create_info.api_version = VK_MAKE_API_VERSION(0, 1, i, 0);
+#else // VULKANSC
+    // Now create an instance and query the functions again - should work because the instance version is less than VKSC 1.1
```

Comment:
VKSC 1.1 comment is confusing.

Reply:
I could change it to something like, "should work because the instance version is valid". I was only trying to match the comment above. It's probably confusing as the comment above is outdated. The above comment would make more sense if 1.2 was replaced with 1.3, as it would then match the source, but it would still be confusing as VK 1.3 is now valid and the source is here is behind ToT

Reply:
Right - at the time this was forked VK 1.3 didn't yet exist.